assets/prototype/domain/shared/predicates/dateUtilities/diff.ts

```diff
@@ -0,0 +1,69 @@
+import {
+	differenceInBusinessDays,
+	differenceInCalendarDays,
+	differenceInCalendarISOWeekYears,
+	differenceInCalendarISOWeeks,
+	differenceInCalendarMonths,
+	differenceInCalendarQuarters,
+	differenceInCalendarWeeks,
+	differenceInCalendarYears,
+	differenceInDays,
+	differenceInHours,
+	differenceInISOWeekYears,
+	differenceInMilliseconds,
+	differenceInMinutes,
+	differenceInMonths,
+	differenceInQuarters,
+	differenceInSeconds,
+	differenceInWeeks,
+	differenceInYears,
```

Comment:
I've added all functions, but we need to decide what we will need.
This will reduce the bundle size:
![image](https://user-images.githubusercontent.com/1477453/70981472-de609200-20bd-11ea-93c2-c4e067e7b6af.png)